kolibri/core/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
```

Comment:
Had only just registered that this was happening during startup only - my assumption is that of the PRAGMAs we are applying below, only 'WAL' will persist, because the others are applied anew at each connection, no? Might explain the minimal impact we saw of the others in the performance tests?